libs/mock-block-dock/src/datastore/use-mock-datastore/temporal.ts

```diff
@@ -155,7 +155,7 @@ export const useMockDatastore = (
               JSON.stringify(currentGraph),
             ) as Subgraph;
 
-            addEntitiesToSubgraphByMutation(newSubgraph, [newEntity]);
+            addEntityVerticesToSubgraphByMutation(newSubgraph, [newEntity]);
```

Comment:
Do we not need to also infer edges now?

Reply:
Hm.. just calling the edges method will be problematic I think. The logic there isn't meant to loop through all existing edges and update their intervals so we may end up with some pretty strange edges, possibly even incorrect ones as we wouldn't be modifying the old intervals to end at the new entity.

I think there's already a bug in `updateEntity` in MBD _in temporal_ datastores so.. I'm not entirely sure what we do here.

Do you think you could spend 10 minutes looking into this to understand the situation and see how easy it'd be to rectify? If it's a lot I think we might just want to throw an error (as opposed to a plain TODO). No-one is using the temporal MBD implementation at the moment to my knowledge, and it somewhat just serves as a sanity check on `tsc` for the output of the graph module so we can continue to use temporal types over in HASH.